content/blog/2019/08/2019-08-17-remoting-kafka-kubernetes-release-2.0.adoc

```diff
@@ -0,0 +1,88 @@
+
+image:/images/post-images/remoting-kafka/cloud-nodes.png[Remoting Kafka Agent get provisioned]
+
+== Helm Chart
+
+Helm chart for Remoting over Apache Kafka plugin is based on link:https://github.com/helm/charts/tree/master/stable/jenkins[stable/jenkins] chart and link:https://github.com/helm/charts/tree/master/incubator/kafka[incubator/kafka] chart. You can follow the instruction link:https://github.com/jenkinsci/remoting-kafka-plugin/blob/master/README.md[here] to install a demo ready-to-use Helm release. Your `kubectl get all` should look like this:
+
+[source, bash]
+----
+NAME                                READY   STATUS    RESTARTS   AGE
+pod/demo-jenkins-64dbd87987-bmndf   1/1     Running   0          2m21s
+pod/demo-kafka-0                    1/1     Running   0          2m21s
+pod/demo-zookeeper-0                1/1     Running   0          2m21s
+
+NAME                              TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                      AGE
+service/demo-jenkins              NodePort    10.108.238.56   <none>        8080:30386/TCP               2m21s
+service/demo-jenkins-agent        ClusterIP   10.98.85.184    <none>        50000/TCP                    2m21s
+service/demo-kafka                ClusterIP   10.109.231.58   <none>        9092/TCP                     2m21s
+service/demo-kafka-headless       ClusterIP   None            <none>        9092/TCP                     2m21s
+service/demo-zookeeper            ClusterIP   10.103.2.231    <none>        2181/TCP                     2m21s
+service/demo-zookeeper-headless   ClusterIP   None            <none>        2181/TCP,3888/TCP,2888/TCP   2m21s
+
+NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/demo-jenkins   1/1     1            1           2m21s
+
+NAME                                      DESIRED   CURRENT   READY   AGE
+replicaset.apps/demo-jenkins-64dbd87987   1         1         1       2m21s
+
+NAME                              READY   AGE
+statefulset.apps/demo-kafka       1/1     2m21s
+statefulset.apps/demo-zookeeper   1/1     2m21s
+----
+
+== How to Contribute
+
+You are welcome to try out the plugin and integrate it into your current setup. If you find out any bug or if you would like to request new feature, you can create ticket at link:https://issues.jenkins-ci.org/browse/JENKINS-58962?jql=project%20%3D%20JENKINS%20AND%20status%20in%20(Open%2C%20%22In%20Progress%22%2C%20Reopened)%20AND%20component%20%3D%20remoting-kafka-plugin[JIRA]. If you would like to contribute code directly, you can create pull requests in the GitHub page below.
+
+== Links
+
+* link:https://youtu.be/tnoObQqGhyM?t=2368[Phase 2 Demo Video]
+* link:https://docs.google.com/presentation/d/1BN2lUcI5UweN2pumAu2m4XHIVXQw6ujzeO9Fbage3ys/edit?usp=sharing[Phase 2 Presentation Slides]
+* link:https://jenkins.io/blog/2019/07/11/remoting-kafka-kubernetes-phase-1/[Phase 1 Blog Post]
```

Comment:
Do you really need the phase 1 demos? Phase 2 covers the topic pretty well, right?

Reply:
bump

Reply:
@oleg-nenashev Phase 2 Demo doesn't have the Start Kafka on Kubernetes feature demonstration